Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: vault
-version: 0.21.0
+version: 0.21.1
```

Comment:
Seems like this should be 0.22.0 since there's a new feature being released?

Reply:
Fair enough. I'll update it.